WordPress/Classes/Models/SiteAddress.swift

```diff
@@ -0,0 +1,5 @@
+/// Models a Site Address
+struct SiteAddress {
```

Comment:
Would it be possible to reuse the existing [`DomainSuggestion`](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/develop/WordPressKit/DomainsServiceRemote.swift#L4) vended by the existing Domains service?